README.rst

```diff
@@ -169,12 +169,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Ziebarth, Malte and
```

Comment:
We might want to add **Yohai Magen** here if #1417 is merged before the release.

Reply:
done in https://github.com/GenericMappingTools/pygmt/pull/1414/commits/acbf3398835d5378f045bf13d5f3e5fd188db425.